src/assets/scripts/aircraft/AircraftController.js

```diff
@@ -353,6 +352,25 @@ export default class AircraftController {
         return this.aircraft.models[icao];
     }
 
+    /**
+     * Remove the specified aircraft from `AircraftController.aircraft`
+     * @method removeAircraftInstanceModelFromList
+     * @param  {Aircraft} aircraft the aircraft to remove
+     */
+    removeAircraftInstanceModelFromList(aircraft) {
+        _pull(this.aircraft.list, aircraft);
```

Comment:
I like that you've done here, however, I would prefer to see `_without` instead because it adds just a little more context.  That and it doesn't mutate the array it actually returns a brand new one.

`this.aircraft.list = _without(this.aircraft.list, aircraft)`

You are very much on the right track!